server/settings/conversion.py

```diff
@@ -15,10 +23,22 @@ def _convert_validate_subset_name(overrides: dict[str, Any]) -> None:
             "ValidateSubsetName"
         )
 
+def _enable_create_render_rops_use_render_product_type(
+        overrides: dict[str, Any]
+) -> None:
+    # Enforce render creators `render_rops_use_render_product_type` to True
+    # to remain backwards compatible with older versions
+    create = overrides.setdefault("create", {})
+    create["render_rops_use_legacy_product_type"] = True
```

Comment:
Is there any way to apply this backward compatible override like this only to project overrides, and not to studio overrides so that the studio default for new project still start of using the NEW default, even when updating from older addons in that particular AYON server?

Reply:
Confirmed by @martastain and @iLLiCiTiT 

There currently isn't.
And:
> tbf: it never occured to me this may be needed since they use the same model